message DseResetLeaderResult
{
	optional int32		result = 1;//返回结果，0=成功，1=失败
}